htdocs/core/class/extrafields.class.php

```diff
@@ -2343,4 +2342,14 @@ public function getOptionalsFromPost($extrafieldsobjectkey, $keyprefix = '', $ke
 
 		return 0;
 	}
+
+	public static function isEmptyValue($v, string $type) {
+		if ($v === null || $v === '') $isEmpty = true;
+		elseif (is_array($v) || $type == 'select') $isEmpty = empty($v);
+		elseif ($type == 'link') $isEmpty = $v == '-1';
```

Comment:
What mean $isEmpty = $v == '-1';
Can you inclue parenthesis so code would be clear ?

Reply:
Parentheses added for clarity.